sdk/turing/version.py

```diff
@@ -1,2 +1 @@
-# Do not update this field manually; this value gets updated automically during build time
```

Comment:
what the motivation for this change?

Reply:
Hmm, I don't remember updating this. I think it is because I do pip install --editable and somehow it got overwritten. Will revert it.